workflows/raredisease.nf

```diff
@@ -403,19 +408,28 @@ workflow RAREDISEASE {
         ).set {ch_snv_annotate}
         ch_versions = ch_versions.mix(ch_snv_annotate.versions)
 
-        ch_snv_annotate.tbi
-            .concat(ANALYSE_MT.out.tbi)
-            .groupTuple()
-            .set { ch_merged_tbi }
-        ch_snv_annotate.vcf_ann
-            .concat(ANALYSE_MT.out.vcf)
-            .groupTuple()
-            .set { ch_merged_vcf }
-        ch_merged_vcf.join(ch_merged_tbi).set {ch_concat_in}
-
-        BCFTOOLS_CONCAT (ch_concat_in)
+        ch_snv_annotate = ANNOTATE_SNVS.out.vcf_ann
+
+        if (params.annotate_mt_switch) {
+
+            ANNOTATE_SNVS.out.tbi
+                .concat(ANALYSE_MT.out.tbi)
+                .groupTuple()
+                .set { ch_merged_tbi }
+
+            ANNOTATE_SNVS.out.vcf_ann
+                .concat(ANALYSE_MT.out.vcf)
+                .groupTuple()
+                .set { ch_merged_vcf }
+
+            ch_merged_vcf.join(ch_merged_tbi).set {ch_concat_in}
+
+            BCFTOOLS_CONCAT (ch_concat_in)
```

Comment:
This step could be troublesome. We need to either remove the existing MT variants from the vcf produced by DV/GLnexus or run bcftools concat with the '-a' option, which I can't see that we do.

Reply:
We are removing the mitochondrial variants with GATK's selectvariants in the call snv subworkflow

Reply:
But I should move it to the raredisease subworkflow and run it just before executing bcftools concat. As it stands now, if one chooses not to run the dedicated mt workflow then the variants from deepvariant will still be removed 😅

Reply:
Ah! Missed that one. 
However, isn't that part always executing? I don't think we should remove the MT variants unless we are running a dedicated workflow.

Reply:
You beat me to it 😆